index.Rmd

```diff
@@ -5,10 +5,10 @@ subtitle: An interactive tool for understanding the distribution of invasive ins
 output: html_document
 ---
 
-# Introduction
+# Introduction 
```

Comment:
This introduction does a good job of laying the groundwork for the project and clearly states the question you want to explore in your analysis.  I do wonder, though, how you will be integrating the temperature increase mentioned in that question.